geometry_msgs/Polygon screen_contour
geometry_msgs/Pose2D screen_pose
float32 hue